### ISSUE TITLE
Analytics Graph is not responsive on mobile devices

### ISSUE THREAD

**Issue by njaffer** [samvera/hyrax#4530]
### Descriptive summary
Analytics Graph is not responsive on mobile devices. Additionally, it is not accessible by screen readers.

### Rationale
When testing Hyrax, nurax on a mobile device, the analytics page has two graphs that do not scale according to the screen size.

### Expected behavior
The graphs should be responsive to the screen size

### Actual behavior
The graphs extends the screen size on a mobile device

### Steps to reproduce the behavior

1. View a work
2. Click on Analytics button

**Comment by no-reply** [comment]
> Additionally, it is not accessible by screen readers.

can we break this out into a separate ticket and tag `accessibility-concern`? i think this should be described and prioritized separately from the responsiveness issue.

**Comment by njaffer** [comment]
yeah sure, I will create another ticket and tag accessibility-concern
